src/confdb/confdb.c

```diff
@@ -649,8 +649,6 @@ int confdb_init(TALLOC_CTX *mem_ctx,
     struct confdb_ctx *cdb;
```

Comment:
Please, change commit subject of `MONITOR: changed startup logic to: ` it does not work well.

Reply:
Done.